README.md

```diff
@@ -27,11 +27,9 @@ Creates a custom permission set SampleDigitalLendingClone.
 Assigns this and several other permission sets to the user.
 </details>
 
-2. Navigate to `Contract Lifecycle Management`'s `General Settings` from Setup (`Feature Settings` &rarr; `Contract Lifecycle Management` &rarr; `General Settings`), and enable `Salesforce Contracts`.
+1. Navigate to `Document Generation`'s `General Settings` from Setup (`Feature Settings` &rarr; `Document Generation` &rarr; `General Settings`), and enable `Design Document Templates in Salesforce`.
 
-3. Navigate to `Document Generation`'s `General Settings` from Setup (`Feature Settings` &rarr; `Document Generation` &rarr; `General Settings`), and enable `Design Document Templates in Salesforce`.
-
-4. Run `sf project deploy start --metadata-dir metadata/OrgSetup -o YourOrgAlias`
+1. Run `sf project deploy start --metadata-dir metadata/OrgSetup -o YourOrgAlias`
```

Comment:
Can you please check the step numbers here. They might need update too

Reply:
In markdown you can put 1. for each line number and it will figure out the correct line number.

https://carpentries-incubator.github.io/markdown-intro/06-lists/index.html

Reply:
Ohh didn't know that, thanks for sharing